hs_file_types/views.py

```diff
@@ -82,8 +83,8 @@ def set_file_type(request, resource_id, hs_file_type, file_id=None, **kwargs):
 
         response_data['status'] = 'success'
         response_data['message'] = msg
-        spatial_coverage_dict = get_coverage_data_dict(res)
-        response_data['spatial_coverage'] = spatial_coverage_dict
+        resource_metadata = get_resource_metadata(res)
```

Comment:
I don't know if this belongs here.  I feel like a better pattern would be to send this metadata over in it's own ajax call.  That way, when we add more functions that require the metadata to be refreshed, the frontend can make the call instead of updating the server to include this data with each call?